contributing.rst

```diff
@@ -0,0 +1,20 @@
+Contributing
+============
+
+Gluster Infrastructure is open to code contributions. Almost all of our
+infrastructure is handled with code. If you cannot contribute to a particular
+piece of infrastructure in our code repos, that's a bug. Please talk to us.
```

Comment:
Should we have a way to contact us listed somewhere ? (like ping on irc, contact us by mail + infra list ?)